src/contract_wrappers/modules/transfer_manager/general_transfer_manager_wrapper.ts

```diff
@@ -178,6 +188,107 @@ interface GetKYCDataParams {
   investors: string[];
 }
 
+/**
+ * @param investor is the address of the investor.
+ * @param flag index of flag to change. flag is used to know specifics about investor like isAccredited.
+ * @param value value of the flag. a flag can be true or false.
+ */
+interface ModifyInvestorFlag extends TxParams {
+  investor: string;
+  flag: FlagsType;
+  value: boolean;
+}
+
+/**
+ * @param investors List of the addresses to modify data about.
+ * @param flag index of flag to change. flag is used to know specifics about investor like isAccredited.
+ * @param value value of the flag. a flag can be true or false.
+ */
+interface ModifyInvestorFlagMulti extends TxParams {
+  investors: string[];
+  flag: FlagsType[];
+  value: boolean[];
+}
+
+/**
+ * @param from Address of the sender
+ * @param to Address of the receiver
+ */
```

Comment:
We don't know what means this data inside SC.

`    /**
     * @notice Default implementation of verifyTransfer used by SecurityToken
     * @param _from Address of the sender
     * @param _to Address of the receiver
    */
    function verifyTransfer(
        address _from,
        address _to,
        uint256 /*_amount*/,
        bytes memory /* _data */
    )`

Reply:
waiting core response

Reply:
"This data can be used by different modules in different ways. The only active module that uses this data is GTM. It can use this data to whitelist investors during the transfer."

what would you write as `data` description?

Reply:
Something like `additional data for module internal use`